shopinvader_customer_activity_date/services/service.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Camptocamp SA (https://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import AbstractComponent
+
+
+class BaseShopinvaderService(AbstractComponent):
+    _inherit = "base.shopinvader.service"
+
+    def dispatch(self, method_name, *args, params=None):
+        res = super().dispatch(method_name, *args, params=params)
+        if self.invader_partner_user:
```

Comment:
this triggers a write on all calls. I would enable/disable it via flags on  the backend

Reply:
added a roadmap entry to make it configurable